src/main/java/com/alipay/oceanbase/rpc/table/ObTableClientQueryImpl.java

```diff
@@ -162,6 +163,7 @@ public ObTableClientQueryStreamResult executeInternal() throws Exception {
                     end[i] = endKey.getObj(i).getValue();
                 }
                 ObBorderFlag borderFlag = rang.getBorderFlag();
+                // result of List<Pair<logicId, obTableParams>>
```

Comment:
Maybe you should state who it is

Reply:
pairs ？

Reply:
Yes, it means that _pairs_ is a list of pair, which left element is the logicId and the right element is the param.